src/main/java/homework_nr_3/HomeWork3.java

```diff
@@ -0,0 +1,65 @@
+package homework_nr_3;
+
+public class HomeWork3 {
+    public static void main(String[] args) {
+
+        int monthNumber = 11;
+
+        switch (monthNumber){
+            case 1:
+            System.out.println("JAN");
+                break;
+            case 2:
+            System.out.println("FEB");
+                break;
+            case 3:
+                System.out.println("MAR");
+                break;
+            case 4:
+                System.out.println("APR");
+                break;
+            case 5:
+                System.out.println("MAY");
+                break;
+            case 6:
+                System.out.println("JUN");
+                break;
+            case 7:
+                System.out.println("JUL");
+                break;
+            case 8:
+                System.out.println("AUG");
+                break;
+            case 9:
+                System.out.println("SEP");
+                break;
+            case 10 :
+                System.out.println("OCT");
+                break;
+            case 11:
+                System.out.println("NOV");
+                break;
+            case 12:
+                System.out.println("DEC");
+                break;
```

Comment:
there's no default here. add default for a case when a number that's outside 1-12 bounds is used